compare_usafacts_nyt_083120.Rmd

```diff
@@ -0,0 +1,427 @@
+  filter(date>=date("2020-04-01")) %>%
+  mutate(difference = NYT-USAFacts,
+         percent_difference = 100*(NYT-USAFacts)/USAFacts) %>%
+  gather(source, deaths, c("NYT","USAFacts","difference","percent_difference")) %>%
+  mutate(panel=case_when(source=="difference"~"Difference",
+                         source=="percent_difference"~"Percent difference",
+                         T~"Cases"),
+         color=case_when(source=="NYT"~"NYT",
+                         source=="USAFacts"~"USAFacts",
+                         T~"Comparison"),
+         color=fct_rev(color)) %>%
+  ggplot(aes(x=date, y=deaths, color=color)) + geom_line() + geom_hline(yintercept=0, linetype="dotted") +
+  facet_grid(panel~., scale="free_y") + 
+  labs(color="") + ylab("") +
+  ggtitle("Comparison of national deaths reported by each source",
+          "April 1st, 2020 onward") +
+  theme_mpr() + 
+  scale_color_mpr(type = "qual", discrete = TRUE)
+  #scale_fill_mpr(type = "qual", discrete = TRUE)
+ggplotly(g, tooltip="y")
+```
+
+## Section 2: Comparison of unallocated cases
+`r last_update_str`
+
+```{r}
+nyt_unknown <- dat_nyt %>% filter(county=="Unknown") %>% filter(date==max(date)) %>% 
+  select(state, cases, deaths) %>% mutate(source="NYT") %>%
+  filter(state!="Guam")
+
+usafacts_unknown_cases <- dat_usafacts_cases %>% filter(fips=="00000") %>% 
+  mutate(val=cases, var="cases") %>%
+  filter(date==max(date)) %>% select(state, var, val)
+
+usafacts_unknown_deaths <- dat_usafacts_deaths %>% filter(fips=="00000") %>% 
+  mutate(val=deaths, var="deaths") %>%
+  filter(date==max(date)) %>% select(state, var, val)
+
+usafacts_unknown <- bind_rows(usafacts_unknown_cases, usafacts_unknown_deaths) %>%
+  spread(var, val) %>% mutate(source="USAFacts") %>%
+  left_join(tibble(state_name = state.name, state = state.abb), by="state") %>%
+  mutate(state=state_name) %>% select(-state_name)
+```
+
+### Unallocated cases and deaths by state 
+This plot shows only states where there are unallocated cases or deaths in either data source.
+
+```{r, fig.height=8, fig.width=10}
+g <- bind_rows(nyt_unknown, usafacts_unknown) %>% filter(cases>0) %>%
+  arrange(desc(cases)) %>% 
+  mutate(state = fct_rev(fct_relevel(as.factor(state), unique(.$state)))) %>%
+  spread(source, cases) %>% mutate_at(c("USAFacts","NYT"), ~ifelse(is.na(.),0,.)) %>%
+  gather("source","cases",c("USAFacts","NYT")) %>%
+  rename(Cases=cases, Deaths=deaths) %>%
+  gather("var","val", c("Cases","Deaths")) %>% 
+  
+  ggplot(aes(x=state, y=val, fill=source)) +
+  geom_col(position = position_dodge()) + 
+  coord_flip() + facet_grid(.~var, scale="free") +
+
+  xlab("State") + ylab("Number of unallocated cases") + labs(fill="") +
+  theme_mpr() + 
+  #scale_color_mpr(type = "qual", discrete = TRUE) +
+  scale_fill_mpr(type = "qual", discrete = TRUE)
+ggplotly(g, tooltip=c("x","y"))
+```
+
+### A closer look at Georgia
+
+#### Unallocated cases in Georgia over time
```

Comment:
these figures also have the legend issue I noted above (I see (NYT,2) and (USAFacts,1) not just NYT and USAFacts)